lib/NetKAT_LocalCompiler.ml

```diff
@@ -250,6 +257,8 @@ let opt_to_table sw_id t =
   in
   let rec next_table_row tests mk_rest t =
     match T.unget t with
```

Comment:
@arjunguha: I have no idea what the semantics of Pipes are supposed to be so I don't know if this makes sense or not.